src/pocketmine/lang/locale/deu.ini

```diff
@@ -324,7 +324,7 @@ pocketmine.plugin.deprecatedEvent=Plugin {%0} hat einen Listener für '{%1}' auf
 pocketmine.plugin.eventError="Event '{%0}' konnte nicht zu '{%1}' weitergegeben werden: {%2} in {%3}"
 
 # -------------------- AdditionalLanguage --------------------
-commands.weather.usage=/weather [clear|rain|thunder] 
+commands.weather.usage=/weather [clear|rain|thunder] (<Welt>)
```

Comment:
Yes. Optional things are in [], necessary arguments are in <>

Und ich meinte ()